app/models/order.rb

```diff
@@ -12,10 +12,17 @@ class Order < ActiveRecord::Base
   scope :completed, -> { where("status = ?", "completed") }
   scope :cancelled, -> { where("status = ?", "cancelled") }
 
+  def buyer
+    User.find(user_id)
+  end
+
+  def sellers
+    order_items.map(&:seller).uniq
+  end
```

Comment:
this way you can find out easily who is the seller user and who is the buyer user on an order